File: my-folder/0001-two-sum/solution.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        for (int i = 0; i < nums.Length; i++) {
            int index = Array.IndexOf(nums, target - nums[i]);
            if (index >= 0 && index != i) {
                return [i, index];
            }
        }
        return null;
    }
}

File: my-folder/0001-two-sum/solution.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    const note = new Map();

    for (let i = 0; i < nums.length; i++) {
        let complement = target - nums[i];

        if (note.has(complement)) {
            return [i, note.get(complement)];
        }

        note.set(nums[i], i);
    }
};
